src/Integration/Exclusions/AnalyzableFileIndicator.cs

```diff
@@ -52,11 +56,17 @@ public bool ShouldAnalyze(string filePath)
 
             if (serverExclusions == null)
             {
+                logger.LogDebug("[AnalyzableFileIndicator] No server settings were found.");
                 return true;
             }
 
             var shouldAnalyze = IsIncluded(serverExclusions, filePath) && !IsExcluded(serverExclusions, filePath);
 
+            logger.LogDebug($"[AnalyzableFileIndicator]" +
+                            $"\n  {serverExclusions}" +
```

Comment:
Relies on this PR that adds ToString https://github.com/SonarSource/sonarqube-webclient-dotnet/pull/115/files